dist/pf-relative.js

```diff
@@ -90,13 +98,28 @@ var noop = function noop () {};
 
 module.exports = function RelativeDate (date) {
   if (!(this instanceof RelativeDate)) return new RelativeDate(date);
-  date = date ? new Date(date) : new Date();
 
+  var lastTimeout = null;
   var listener = noop;
+
   this.onTick = function onTick (newListener) {
     listener = newListener;
   };
 
+  this.setDate = function setDate (newDate) {
+    date = newDate ? new Date(newDate) : new Date();
+    clearTick();
+    tick();
+  };
+
+  function clearTick () {
+    if (lastTimeout) {
```

Comment:
I _think_ that there's no need for this check here. AFAIR, `clearTimeout(null)` does not break.